source/javascripts/components/StepItem/StepItem.tsx

```diff
@@ -12,60 +12,70 @@ import verifiedIcon from "../../../images/step/badge-verified.svg";
 import deprecatedIcon from "../../../images/step/badge-deprecated.svg";
 
 type StringProps = {
-    alwaysLatest: string
-}
+	alwaysLatest: string;
+};
 
 type StepItemProps = {
-  step: Step,
-  strings: StringProps,
-  selected: boolean,
-  highlightVersionUpdate: boolean,
-  stepIndex: number,
-  onSelected: (step: Step, index: number) => void
-}
+	step: Step;
+	title: string;
+	version: string;
+	strings: StringProps;
+	selected: boolean;
+	highlightVersionUpdate: boolean;
+	stepIndex: number;
+	onSelected: (step: Step, index: number) => void;
+};
 
 export const normalizeIconUrl = (step?: Step): string | undefined => {
-  if (!step) {
-    return;
-  }
+	if (!step) {
+		return;
+	}
 
-  const stepIconURL = step.iconURL();
-  return stepIconURL || defaultStepIcon;
+	const stepIconURL = step.iconURL();
+	return stepIconURL || defaultStepIcon;
 };
 
-const tabIndex = (selected: boolean): number => selected ? -1 : 0;
+const tabIndex = (selected: boolean): number => (selected ? -1 : 0);
 
-const stepVersion = (step: Step, highlightVersionUpdate: boolean) => (highlightVersionUpdate
-  ? <Text>{step.version}</Text>
-  : <Badge backgroundColor="red-3" color="white">
-      <Icon name="ArrowUp" />
-      {step.version}
-    </Badge>
-);
+const stepVersion = (step: Step, highlightVersionUpdate: boolean) =>
+	highlightVersionUpdate ? (
+		<Text>{step.version}</Text>
+	) : (
+		<Badge backgroundColor="red-3" color="white">
+			<Icon name="ArrowUp" />
+			{step.version}
+		</Badge>
+	);
```

Comment:
can we use linter for this? It's not in the scope, just asking :)

Reply:
Yup, let's!